Nosach/web-quiz-project/src/main/java/servlet/QuizAddServlet.java

```diff
@@ -0,0 +1,32 @@
+package servlet;
+
+import service.QuizService;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class QuizAddServlet extends HttpServlet {
+
+    QuizService ts = QuizService.QUIZ_SERVICE;
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+        boolean formChecked = Boolean.parseBoolean(req.getParameter("formChecked"));
```

Comment:
Why do you need this formChecked parameter?

Reply:
Request enters into AddServlet in two cases: 1. User clicked 'add' button on main  page, 2. User inserted data at "adding quiz page" into fields and clicked "submit".   formChecked parameter shows did we enter with 1st case ("false") or 2d ("true").